graylog2-server/src/main/java/org/graylog/security/shares/DefaultGranteeService.java

```diff
@@ -80,12 +93,20 @@ public DefaultGranteeService(UserService userService, GRNRegistry grnRegistry, G
                 .collect(ImmutableSet.toImmutableSet());
     }
 
+    private ImmutableSet<AvailableGrantee> getGlobalGrantees() {
+        final ImmutableSet.Builder<AvailableGrantee> builder = ImmutableSet.<AvailableGrantee>builder();
+        final Optional<UserAndTeamsConfig> config = Optional.ofNullable(clusterConfigService.get(UserAndTeamsConfig.class));
+        if (config.isPresent() && config.get().sharingWithEveryone()) {
```

Comment:
This would not add the `everyone` grantee if the clusterconfig has never been saved.
(but see above suggestion on how to avoid that problem)